core/cardano-sl-core.cabal

```diff
@@ -148,10 +148,9 @@ library
 
   build-depends:       QuickCheck
                      , aeson
-                     , ansi-terminal
+                     , ansi-terminal    
```

Comment:
😒

Reply:
My bad, I was using vim to edit those configs, I only have trailing space hook in emacs.